.sync/azure_pipelines/MuDevOpsSync.yml

```diff
@@ -0,0 +1,90 @@
+## @file
+# Azure Pipeline build file for a build using mu_devops.
+#
+# NOTE: This file is automatically synchronized from Mu DevOps. Update the original file there
+#       instead of the file in this repo.
+#
+# - Mu DevOps Repo: https://github.com/microsoft/mu_devops
+# - File Sync Settings: https://github.com/microsoft/mu_devops/blob/main/.sync/Files.yml
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+{% import '../../../Version.njk' as sync_version -%}
+
+resources:

```

Comment:
@makubacki , would be curious to get your input on this. This file exists pretty much to get the devops version. Kun explored a number of ways to do this without having to sync a wrapper template, but it doesnt seem practical. Any thoughts on how this could be done?

Reply:
If I understand correctly, I don't think so.

1. The `resources` section is processed very early to set up the pipeline and template binding.
2. We decided to fix repos on an exact version of mu_devops so that needs to be recorded in the repo's history.

I believe (1) eliminates most dynamic options and (2) means a sync of the mu_devops version to the repo (of some kind) is needed.

As the scripts are currently written, I think this route is acceptable to reduce number of files synced and still control the version in one location (`Version.njk`).